lib/simplifyExpression/collectAndCombineSearch/multiplyLikeTerms.js

```diff
@@ -141,13 +146,12 @@ function addOneExponent(node) {
 // e.g. 10^2 * 10^3 -> 10^(2+3)
 // Returns a Node.Status object.
 function collectConstantExponents(node) {
-  const constantTermList = node.args.map(n => new checks.canMultiplyLikeTermConstantNodes.createPowerNode(n));
   // If we're multiplying constant nodes together, they all share the same
   // base. Get that from the first node.
-  const baseNode = constantTermList[0].base;
+  const baseNode = checks.canMultiplyLikeTermConstantNodes.getBaseNode(node.args[0]);
```

Comment:
Yeah, I also think that making a new module is the way. Because it will also be used for when dividing power terms, so a new module will be neater than having the same code in two different modules.